src/platform/mach.js

```diff
@@ -200,25 +200,33 @@ const mach = {
     });
 
     canvas.addEventListener("keyup", (ev) => {
-      mach.events.push(...[2, convertKeyCode(ev.code)]);
+      mach.events.push(...[3, convertKeyCode(ev.code)]);
```

Comment:
Would love to understand why these are changing. Probably I've just missed something in this PR.

Reply:
Same as above - there is a new key_repeat event to match glfw.